src/store/mutators.ts

```diff
@@ -412,16 +412,20 @@ export const removeNode = (
       }
     }
 
-    const exitIdx = getExitIndex(fromNode.node, fromExitUUID);
-    updatedNodes = mutate(updatedNodes, {
-      [fromNodeUUID]: {
-        node: {
-          exits: {
-            [exitIdx]: { destination_uuid: set(destination) }
+    try {
+      const exitIdx = getExitIndex(fromNode.node, fromExitUUID);
+      updatedNodes = mutate(updatedNodes, {
+        [fromNodeUUID]: {
+          node: {
+            exits: {
+              [exitIdx]: { destination_uuid: set(destination) }
+            }
           }
         }
-      }
-    });
+      });
+    } catch {
+      console.log('The relation has been already deleted.');
```

Comment:
I think for this case we can have the catch statement doing nothing

Reply:
I've added requested changes